packages/base/room.gts

```diff
@@ -841,6 +921,7 @@ export interface ReactionEventContent {
     key: string;
     rel_type: 'm.annotation';
   };
+  result?: any;
```

Comment:
This is the main key that will fuel pass search results so that we can create a new card